repo/fsrepo/migrations/ipfsfetcher/ipfsfetcher.go

```diff
@@ -277,3 +281,50 @@ func parsePath(fetchPath string) (ipath.Path, error) {
 	}
 	return ipfsPath, ipfsPath.IsValid()
 }
+
+func readIpfsConfig(repoRoot *string) (bootstrap []string, peers []peer.AddrInfo) {
+	if repoRoot == nil {
+		return
+	}
+
+	cfgPath, err := config.Filename(*repoRoot)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		return
+	}
+
+	cfgFile, err := os.Open(cfgPath)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		return
+	}
+	defer cfgFile.Close()
+
+	// Attempt to read bootstrap addresses
+	var bootstrapCfg struct {
+		Bootstrap []string
+	}
+	err = json.NewDecoder(cfgFile).Decode(&bootstrapCfg)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "cannot read bootstrap peers from config")
+	} else {
+		bootstrap = bootstrapCfg.Bootstrap
+	}
+
+	if _, err = cfgFile.Seek(0, 0); err != nil {
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
Added the comment:
```go
        // If Seek fails, only log the error and continue on to try to read the
        // peering config anyway as it might still be readable
```